### ISSUE TITLE
Rename Issue 

### ISSUE THREAD

**Issue by Seanxedge** [Blossomforth/BlossomsPokemonGoManager#562]
Went and renamed all my poke to their original name since I had some nicknames and previous IVs added into their names. I proceeded to rename them again to be  name_IV% but I keep getting the same error Skipped renaming Xatu, already named ""
Not sure if a common issue or problem but I haven't been able to solve it. Going to wipe and rerun the program and give it a try in the morning.

**Comment by Seanxedge** [comment]
Any known fix for this or are we waiting on an update?

**Comment by kribjo** [comment]
Seems related to https://github.com/Wolfsblvt/BlossomsPokemonGoManager/issues/560 ?

**Comment by kribjo** [comment]
I've been trying to reproduce this (in v0.1.7) without success.
Someone else?
I've been trying first multi-renaming to "" and afterwards both with "name_IV%" (sic) as written in the issue report and with "%name%_%iv_rating%" as I think was the intended exact rename syntax. Also tried many other variations on the procedure.

However, during testing, I got another error where one of the renamed Pokémons (the only Murkrow) disappeared from the listing. Refresh list did not fix it, but application restart seemed to do.

**Comment by Grotarin** [comment]
There seem to be somethng with pokemons type.
I try this pattern: %type_1%%type_2%%move_type_1%%dps_1%%move_type_2%%dps_2%
And for every pokemon I get "PoPo" followed by the correct data for the rest.
Ex: a Clefairy gets PoPops11FA13 instead of Fa__ps11FA13.
Is it only me?

**Comment by just-Addict** [comment]
@Grotarin 

First things first, even though it's a renaming issue, your problem is not necessarily the same or even related to the problem of the OP who created this issue. Only comment in an issue with a problem you encounter if that actually relates to the same issue, otherwise it's better to post it as a new issue.  You were lucky I read all the comments but this might easily have gotten overlooked as not being related to the original issue.

> I try this pattern: %type_1%%type_2%%move_type_1%%dps_1%%move_type_2%%dps_2%
> And for every pokemon I get "PoPo" followed by the correct data for the rest.

I can confirm this and also think I found what happens. If I am not mistaken then lines 325 and 332 in `PokeHandler.java` are the issue

Instead of just
`p.getSettings().getType().toString();` and `p.getSettings().getType2().toString();`
I think both statements should lose the `.toString()` bit and then be encapsulated in a 
`PokemonUtils.formatType( ... )` call because the result may still have the prefix **pokemon_type_** and possibly **none** for the second type, which would explain why you get **PoPo** with the used pattern.

**Comment by just-Addict** [comment]
@Seanxedge what is the exact pattern you used?

Like @kribjo I cannot reproduce this in the latest release version. To get the form you mentioned I used the pattern **%name%_%iv_rating%%**
